### ISSUE TITLE
Unable to install pysiaf in astroconda environment

### ISSUE THREAD

**Issue by jotaylor** [spacetelescope/pysiaf#69]
Pysiaf does not install correctly when starting from an astroconda environment. Using conda version 4.5.11 I created a completely new astroconda environment with python 3.6 from the stable channel (http://ssb.stsci.edu/astroconda):
`conda create -n myenv stsci python=3.6`

By default, lxml is installed like so:
```
> conda list lxml
# Name                    Version                   Build  Channel
lxml                      4.2.5            py36hef8c89e_0
```
Doing a `pip install pysiaf` gives me the following versions of lxml and pysiaf:
```
# Name                    Version                   Build  Channel
pysiaf                    0.2.4                     <pip>
lxml                      4.2.5            py36hef8c89e_0  
```
But when importing pysiaf I get the error warned about in the pysiaf README:
```
Reason: Incompatible library version: etree.cpython-36m-darwin.so requires version 12.0.0 or later, but libxml2.2.dylib provides version 10.0.0
```
I follow the instructions to uninstall and install lxml version 3.6.4. Now I get
```
# Name                    Version                   Build  Channel
pysiaf                    0.2.4                     <pip>
lxml                      3.6.4                     <pip>
```
as expected.

Yet still pysiaf won't import, and throws the same error as before.

The only solution I have found is to uninstall *both* pysiaf and lxml and then install pysiaf. When I do this I get identical versions to the original attempt:
```
# Name                    Version                   Build  Channel
pysiaf                    0.2.4                     <pip>
lxml                      4.2.5                     <pip>
```
with the only difference being that lxml is now coming from pip and not conda. For some reason pysiaf imports fine like this. I really don't understand how the pip vs. conda versions of lxml 4.2.5 are different. I can't tell if this actually solved some underlying problem or is just a surface-level fix.

Additionally, it should be noted that doing a `conda install pysiaf` does not give you the same version of pysiaf as `pip install pysiaf`. `conda install` gives you 0.1.8 while `pip install` gives you 0.2.4...

**Comment by Johannes-Sahlmann** [comment]
Thank you for this report. I will look into this. 

Concerning the last comment on conda versus pypi, this is a known issue, see #42

**Comment by jdavies-st** [comment]
When you do

```
conda create -n myenv stsci python=3.6
```

you are creating a conda env that has python installed plus a ton of other packages listed in the `stsci` metapackage.  This includes `lxml`.  And it is installed as a conda package, which means only conda can manage it, not pip.

Then when you `pip install pysaif`, it also needs `lxml`, and this a problem, because you now have 2 package managers arguing over who should manage a single package.  Not good.

The solution is to not install the `stsci` metapackage.  Instead make a new env with only python installed:

```
conda create -n myenv python
```

and then `pip install pysiaf` after activating.  If you need the `stsci` metapackage for some functionality, create a separate conda env with it installed and switch back and forth between envs.

So this is not a bug in `pysiaf`, but a problem with using conflicting package managers for the same installed package, something that Anaconda explicitly recommends against doing.  Read more here

https://www.anaconda.com/blog/using-pip-in-a-conda-environment

**Comment by jotaylor** [comment]
@jdavies-st makes sense. I think the installation instructions 3 years ago were rather different, hence my convoluted methodology :) probably fine to close this issue now.
